perllib/FixMyStreet/Cobrand/Sutton.pm

```diff
@@ -172,4 +172,39 @@ sub garden_cc_check_payment_status {
 sub bulky_collection_time { { hours => 6, minutes => 0 } }
 sub bulky_cancellation_cutoff_time { { hours => 6, minutes => 0 } }
 
+sub bulky_allowed_property {
+    my ($self, $property) = @_;
+
+    return $self->bulky_enabled
+        && bulky_permitted_property_types()->{ $property->{type}{Id} };
+}
+
+sub bulky_permitted_property_types {
+    return {
+        1   => 'Detached',
```

Comment:
This list should not be hard-coded in the code. Please change to use a configuration value, which we already do for general list of `address_types` available to be looked up in waste. `bulky_address_types` for example. Then `bulky_allowed_property` can look up if the value exists in the configured list. We don't need to store the name associated with each address type, that's never used.